modules/manage/pages/kubernetes/k-rolling-restart.adoc

```diff
@@ -0,0 +1,107 @@
+= Perform a Rolling Restart of Redpanda in Kubernetes
+:description: Learn how to perform a rolling restart of your Redpanda cluster when it's running in Kubernetes.
+:rolling-restart:
+:page-context-links: [{"name": "Linux", "to": "manage:cluster-maintenance/rolling-restart.adoc" },{"name": "Kubernetes", "to": "manage:kubernetes/k-rolling-restart.adoc" } ]
+
+A rolling restart involves restarting one broker at a time, while the remaining brokers in your cluster continue running. This is to minimize downtime during a full cluster restart.
+Perform a rolling restart during operations such as configuration updates that require a restart, version upgrades or cluster maintenance.
+
+When you run Redpanda in Kubernetes with the Redpanda Operator or the Redpanda Helm chart, your Redpanda cluster is managed as a StatefulSet where each broker runs inside its own Pod. As a result, you can perform a rolling restart using a single command with the Kubernetes API.
+
+== Prerequisites
+
+You must have the following:
+
+* xref:deploy:deployment-option/self-hosted/kubernetes/index.adoc[A Redpanda cluster running in Kubernetes].
+
+include::upgrade:partial$rolling-upgrades/restart-impact.adoc[leveloffset=+1]
+
+== Perform a rolling restart
+
+A rolling restart involves running the following procedure on each broker, one at a time. The Redpanda Operator and the Redpanda Helm chart automate this procedure.
```

Comment:
Can we link to resources on the Redpanda Operator and Redpanda Helm chart for a quick cross-reference?